src/structlog/dev.py

```diff
@@ -694,7 +693,7 @@ def add_meaningless_arg(arg: str) -> None:
             Column("logger_name", logger_name_formatter),
         ]
 
-    def _repr(self, val: Any) -> str:
+    def value_repr(self, val: Any) -> str:
```

Comment:
I don't think we need to make this method public?

Reply:
OK